.env-regtest-local-sample

```diff
@@ -0,0 +1,78 @@
+###############################################################
+# Sparkswap Environment File
+#
+# These settings control the underlying functionality of sparkswapd and the
+# associated lightning implementations (engines).
+#
+###############################################################
+
+# Set the network that sparkswap and engines will work on.
+NETWORK=regtest
+
+# Set secure credentials for your sparkswap daemon
+RPC_USER=sparkswap
+RPC_PASS=sparkswap
+
+# Relayer Host Addresses for RegTest
+RELAYER_RPC_HOST=host.docker.internal:28492
+
+# Bitcoin Settings
+#
+# We have provided default username/passwords for all active engines (ltc, btc)
+# on the daemon, HOWEVER, if you are self-hosting these files OR running w/ real money
+# then it is highly recommended that you change these defaults for your security
+#
+# BTC_RPC_HOST=your.btc.rpc.host
+BTC_RPC_USER=sparkswapbtc
+BTC_RPC_PASS=sparkswapbtc
+
+# Block Broadcast Address
+# BTC_ZMQPUBRAWBLOCK=your.bitcoin.node.block.address
+
+# Transaction Broadcast Address
+# BTC_ZMQPUBRAWTX=your.bitcoin.node.transaction.address
+
+# Regtest Host
+BITCOIND_CONNECT_HOST=host.docker.internal
+
+# Litecoin Settings
+#
+# We have provided default username/passwords for all active engines (ltc, btc)
+# on the daemon, HOWEVER, if you are self-hosting these files OR running w/ real money
+# then it is highly recommended that you change these defaults for your security
+#
+# LTC_RPC_HOST=your.ltc.rpc.host
+LTC_RPC_USER=sparkswapltc
+LTC_RPC_PASS=sparkswapltc
+
+# Block Broadcast Address
+# LTC_ZMQPUBRAWBLOCK=your.litecoin.node.block.address
+
+# Transaction Broadcast Address
+# LTC_ZMQPUBRAWTX=your.litecoin.node.transaction.address
+
+# Regtest Host
+LITECOIND_CONNECT_HOST=host.docker.internal
+
+# External IP Addresses
+#
+# In order for your lightning nodes to be found by other users on the network
+# you must set an external address for each chain. If you are hosting all containers
+# on the same machine, then these will be the same.
+#
+# This address can either be a TLD or an IP address. Do NOT include the port number
+# as this is defined in the docker-compose file instead.
+#
+EXTERNAL_BTC_ADDRESS=host.docker.internal
+EXTERNAL_LTC_ADDRESS=host.docker.internal
+
+# Path to rpc certs and identity keys
+SECURE_PATH=~/.sparkswap/secure
+
+# Docker compose files
+# docker-compose by default will look for the docker-compose.yml file and apply the docker-compose.override.yml file if
+# it exists. If we specify the COMPOSE_FILE variable, all compose files joined by the COMPOSE_PATH_SEPARATOR will
+# be applied and used
+#
+COMPOSE_PATH_SEPARATOR=:
+COMPOSE_FILE=docker-compose.yml:docker-compose.bitcoind.yml:docker-compose.litecoind.yml:docker-compose.regtest.yml:docker-compose.local.yml
```

Comment:
shouldn't we have an override file at the end of this?

Reply:
see https://github.com/sparkswap/broker/pull/415#discussion_r260016215